feature-libs/asm/tsconfig.lib.json

```diff
@@ -0,0 +1,29 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/lib",
+    "target": "es2015",
+    "module": "es2020",
+    "moduleResolution": "node",
+    "declaration": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "types": [],
+    "lib": ["es2020", "dom"],
+    "paths": {
+      "@spartacus/core": ["dist/core"],
+      "@spartacus/storefront": ["dist/storefrontlib"]
+    }
+  },
+  "angularCompilerOptions": {
```

Comment:
Can we also set
`strictTemplates` to true and check if we can keep it?

Reply:
Just an idea, maybe we can create a new example app with the --strict flag as described in https://angular.io/guide/strict-mode and use it as a guide to improve our tsconfig stricness?